cmd/bugzilla-backporter/main.go

```diff
@@ -92,6 +89,7 @@ func gatherOptions() (options, error) {
 	fs.StringVar(&o.logLevel, "log-level", "info", "Level at which to log output.")
 	fs.StringVar(&o.address, "address", ":8080", "Address to run server on")
 	fs.DurationVar(&o.gracePeriod, "gracePeriod", time.Second*10, "Grace period for server shutdown")
+	fs.StringVar(&o.pluginConfig, "plugin-config", "/etc/plugins/plugins.yaml", "Path to plugin config file.")
```

Comment:
Can you create a pr against openshift/release that adds a mount for this so we can merge this pr and the required config change together?

Reply:
I created this PR for that, is this what you meant? https://github.com/openshift/release/pull/10021